modules/oe_theme_content_publication/src/Plugin/ExtraField/Display/PublicationDescription.php

```diff
@@ -92,6 +92,15 @@ public function viewElements(ContentEntityInterface $entity) {
 
     $cacheability = CacheableMetadata::createFromRenderArray($build);
     $cacheability->addCacheableDependency($media);
+
+    // Run access checks on the media entity.
+    $access = $media->access('view', NULL, TRUE);
+    $cacheability->addCacheableDependency($access);
+    if (!$access->isAllowed()) {
+      $cacheability->applyTo($build);
+      return [];
```

Comment:
```suggestion
     return $build;
```
Otherwise body field value is absent when media is unpublished.